### PR TITLE
Xplat cleanup

### PR DESCRIPTION
**Pull request by david-puglielli**

### CODE REVIEW

File: source/shared/xplat.h [hunk 1]
```diff
@@ -162,59 +85,19 @@ using std::max;
...
-typedef unsigned __int64 windowsULongLong_t;
-#else // *nix (!WwoWH)
 #include <stdint.h> // Use standard bit-specific types (signed/unsigned integrated)
 typedef int32_t windowsLong_t;
 typedef uint32_t windowsULong_t;
 typedef int64_t windowsLongLong_t;
 typedef uint64_t windowsULongLong_t;
-#endif
-typedef windowsLong_t LONG, *PLONG, *LPLONG;
+typedef windowsLong_t LONG;
```
Comment (yitam) [inline source/shared/xplat.h]: so I suppose we don't need *PLONG or *LPLONG?

File: source/shared/xplat.h [hunk 2]
```diff
@@ -234,162 +117,48 @@ typedef windowsULongLong_t ULONGLONG;
 // source code shared with Windows SNAC (which is ODBC 3.8) compiles.
 #define ODBCVER 0x0380
```
Comment (yitam) [inline source/shared/xplat.h]: Do we need to define ODBCVER?

File: source/shared/xplat.h (hunk 1 above)
Comment (david-puglielli) [inline source/shared/xplat.h]: They're not used anywhere in the driver code, so right now those typedefs are not needed.

File: source/shared/xplat.h (hunk 2 above)
Comment (david-puglielli) [inline source/shared/xplat.h]: Actually no, I'll remove it.

Comment (ulvii) [inline source/shared/xplat.h]: Please be careful when removing ODBCVER. We still should be able to compile the drivers without having to specify --with-odbcver option

Comment (yitam) [inline source/shared/xplat.h]: I believe in appveyor it configures without that odbcver option already, so we will see

Comment (ulvii) [inline source/shared/xplat.h]: its not about appveyor. PECL build does not use --with-odbcver configure option.

**Pull request merged**